### PR TITLE
Corrections for m04

### PR DESCRIPTION
**Pull request by alex-lis-dev**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -29,17 +30,23 @@ function App() {
 
   const [isOpen, setOpen] = useState(false);
   const [isDeleteOpen, setIsDeleteOpen] = useState(false);
-  
-  const toggleDialog = () => setOpen(!isOpen);
-  const toggleDeleteDialog = () => setIsDeleteOpen(!isDeleteOpen);
+
+  const toggleDialog = () => {
+    if (isOpen) setMovieToEdit();
```
Comment (hicugi) [inline src/App.jsx:35]: Bad approach... to many thing to be aware of.
It's better to separate create & edit dialogs

File: src/App.jsx [hunk 2]
```diff
@@ -29,17 +30,23 @@ function App() {
...
+    setOpen(!isOpen);
+  };
+  const toggleDeleteDialog = () => {
+    if (isDeleteOpen) setMovieToDelete();
```
Comment (hicugi) [inline src/App.jsx:39]: Not necessary - you are using this modal only for one purpose

File: src/components/GenreSelector/GenreSelector.jsx [hunk 3]
```diff
@@ -16,7 +16,7 @@ const GenreSelector = ({ genres, selectedGenre = "All", onSelect }) => {
           style={{
             backgroundColor: "#232323",
             color: "#FFFFFF",
-            "border-bottom":
+            borderBottom:
               "3px solid " + (genre === currentGenre ? "#F65261" : "#424242"),
```
Comment (hicugi) [inline src/components/GenreSelector/GenreSelector.jsx:20]: Don't use condition inline styling, bad for perfomance. User className toggle

Comment (alex-lis-dev) [reply src/components/GenreSelector/GenreSelector.jsx:20]: fixed in react-mentoring-lis-m05 branch

**Pull request merged**